src/index.js

```diff
@@ -12,9 +12,7 @@ export { default as VmsPageHeader } from './components/VmsPageHeader'
 
 export * from './helpers'
 
-// TODO: consider prefixes or wrapping in a class, split & directory structure
-export * from './vm-actions'
```

Comment:
The action creators shall be accessible from outside.
What about wrapping them in similar way as in case of the reducers below?